Gemfile

```diff
@@ -131,6 +131,8 @@ source "https://rails-assets.org" do
   gem "rails-assets-sprintf",                         "~>1.0.3"
   gem "rails-assets-numeral",                         "~>1.5.3"
   gem "rails-assets-slickgrid",                       "~>2.1.0"
+  gem "rails-assets-angular-patternfly",              "=2.2.0"
+  gem "rails-assets-angular-animate",                 "=1.4.5"
```

Comment:
But I think the version string for angular animate should be the same as for angular, to prevent future mismatches. (and group them together?)